src/MassTransit/MassTransit.csproj

```diff
@@ -30,6 +30,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="AWSSDK.RDS" />
```

Comment:
Yeah, this isn't going to be added to MT proper.

Reply:
Correct, no way.